app/(tabs)/other/valid/index.tsx

```diff
@@ -34,7 +37,10 @@ export default function ValidScreen() {
       }
     >
       <SearchBar query={query} search={search} cancel={cancel} autoFocus />
-      <p>{results}</p>
```

Comment:
`<p>` will cause a crash on mobile

Reply:
This was replaced with a variable number of react-native Text elements